### PR TITLE
feat(docker): how to run bonita container interactively

### PR DESCRIPTION
**Pull request by tnguyen1**
* completing community question https://community.bonitasoft.com/node/35664

### CODE REVIEW

**Review by educhastenier — APPROVED** [review]
👌🏼

**Comment by marielle-spiteri** [comment]
- Could you explain how to stop Bonita without running bonita container interactively? 
- Could you explain how to pass " -e HTTP_API=true -e REST_API_DYN_AUTH_CHECKS=false' when running interactively? 
- There is a typo in the command line:  docker run --name bonita -h <hostname> -v ~/bonita-lic/:/opt/bonita_lic/ -it -p 8080:8080 quay.io/bonitasoft/bonita-subscription:${varVersion}.0 bash
- the command with ${varVersion}.0 doesn't work. Please remove the .0 after ${varVersion}
- Could you add a stop section, the information is not visible but seems lost at the end of a sentence in a section that has the title "Start the container interactively" (...press `Ctrl-C`....). Since we have "start" sections in this page, logically I'm expecting "stop/shutdown" sections, the same way there as here "Starting and shutting down Tomcat" : https://documentation.bonitasoft.com/bonita/7.11/tomcat-bundle

**Comment by tnguyen1** [comment]
retest this please

**Comment by tnguyen1** [comment]
That's how it will be rendered when `${varVersion}.0` is evaluated on the doc site:
image

@mariellebonita I'll let R&D decide how to integrate the latest feedbacks, or you may directly contribute to this PR eventually.

**Comment by educhastenier** [comment]
> * the command with ${varVersion}.0 doesn't work. Please remove the .0 after ${varVersion}

@mariellebonita I confirm that what @tnguyen1 wrote is correct:

image

File: md/bonita-docker-installation.md [hunk 1]
```diff
@@ -57,6 +57,19 @@ This will start a container running the Tomcat Bundle with Bonita Engine + Porta
...
 You can access the portal on http://localhost:8080/bonita and login using the default credentials : install / install
 
+### Start the container interactively
+
+You may also run the Bonita container with an interactive shell session in order to control start and stop of the Tomcat bundle. Although the main usage of the Docker image is to automatically start the Bonita runtime, manually start and stop Tomcat may be useful for development or troubleshooting purposes.
+
+Run the container as follows in order to get an interactive `bash` shell session:
+
+```
+docker run --name bonita -h <hostname> -v ~/bonita-lic/:/opt/bonita_lic/ -it -p 8080:8080 quay.io/bonitasoft/bonita-subscription:${varVersion}.0 bash
```
Comment (tbouffard) [inline md/bonita-docker-installation.md:72]: ⚠️ the license folder is not required by the community image

Comment (educhastenier) [reply md/bonita-docker-installation.md:72]: Correct.
@tnguyen1 Can you provide a second example for Community version?

**Comment by tnguyen1** [comment]
retest this please

File: md/bonita-docker-installation.md (hunk 1 above)
Comment (tnguyen1) [reply md/bonita-docker-installation.md:72]: Just added the command to run for Community version.

**Pull request merged**